docs/administration-guide/20-custom-maps.md

```diff
@@ -0,0 +1,25 @@
+## Custom Maps
+
+By default, Metabase uses OpenStreetMaps for map visualizations, but there are a few customization options.
+
+### Finding Map Settings
+
+To find the map settings:
+
+1. Click the gear icon in the upper right.
+2. Select `Maps` from the navigation on the left.
+
+![Map Settings](images/MapSettings.png)
+
+### Map Tile Server
+
+If you'd like to use a map tile server other than OpenStreetMaps, simply add the map tile server URL to the first field on the page.
+
+### Custom Maps
+
+Metabase also provides the option to add custom regional maps via GeoJSON. To get started, the `Add a map` button. A modal will appear. You will need to enter the name you'd like to use for the map, the URL for the GeoJSON file and the properties that you'd like to use to specify your region's identifier and display name.
```

Comment:
Feels like we might want to give an example here. Like, I'm not 100% sure what filling out "the properties that you'd like to use to specify your region's identifier and display name" entails.